services/order.py

```diff
@@ -0,0 +1,29 @@
+from db.models import Order
+from db.models import User
+from db.models import Ticket
+from django.db import transaction
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(tickets: list, username: str, date: str = None) -> Order:
```

Comment:
The `date` parameter should be of type `datetime` instead of `str` to match the expected data type for `DateTimeField` in Django models.